src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,54 @@
+// #region imports
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+// #endregion
+
+export const PeoplePage = () => {
+  // #region hooks
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsError(false);
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setIsError(true);
+        setIsLoading(false);
+      })
+      .finally(() => setIsLoading(false));
```

Comment:
There is a logical issue here. The `setIsLoading(false)` is called both in the `catch` block and in the `finally` block. This means that `setIsLoading(false)` will be called twice if there is an error. You should remove `setIsLoading(false)` from the `catch` block since the `finally` block will always execute after the `then` or `catch`.